etl/steps/data/garden/ihme_gbd/2024-06-10/leading_causes_deaths.py

```diff
@@ -0,0 +1,151 @@
+from typing import List
+
+import owid.catalog.processing as pr
+from owid.catalog import Table
+from owid.catalog.utils import underscore
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+# naming conventions
+paths = PathFinder(__file__)
+log = get_logger()
```

Comment:
you can use `paths.log` instead of `log`. It works the same way, but it already prints the name of the step.

e.g. `paths.log.info("start")` instead of `log.info("step.start")`